src/Browscap/Data/DataCollection.php

```diff
@@ -467,7 +467,8 @@ public function addSourceFile($src)
             $userAgents,
             (boolean) $divisionData['lite'],
             (boolean) $divisionData['standard'],
-            $versions
+            $versions,
+            substr($src, strpos($src, 'resources/'))
```

Comment:
Normalizing the filename a bit so that everything before "resources/" is removed.